src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private static final Colors[] colors = Colors.values();
+    private static final Random random = new Random();
+
+    public static Colors getRandomColor() {
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/140905715/95e3705b-4af6-4fe2-9839-8e017b44a4de)

Reply:
Same in `Lottery` class